shared/logging_module.rb

```diff
@@ -14,16 +37,23 @@ def logger
             class_name = self.name.split("::").last
           end
 
+          progname ||= class_name
```

Comment:
When using the logger's block syntax, and providing a `progname` we should preserve the expected behavior of having that appear in the log line.

Reply:
🏆